WebService/v1/Controllers/StatusController.cs

```diff
@@ -41,19 +45,42 @@ public sealed class StatusController : Controller
             var errors = new List<string>();
 
             // Check access to Azure IoT Hub
-            var iotHubStatus = await this.devices.PingRegistryAsync();
-            if (!iotHubStatus.Item1)
+            Tuple<bool, string> iotHubStatus = null;
+            try
+            {
+                if (this.IsHubConnectionStringConfigured())
+                {
+                    iotHubStatus = await this.devices.PingRegistryAsync();
+                    if (!iotHubStatus.Item1)
+                    {
+                        statusIsOk = false;
+                        errors.Add("Unable to use Azure IoT Hub service");
+                    }
+                }
+                else
+                {
+                    iotHubStatus = new Tuple<bool, string>(false, "not configured");
```

Comment:
nit: "IoTHub connection string not configured"

Reply:
👍